masurca/4.0.9/README.md

```diff
@@ -0,0 +1,20 @@
+# masurca container
+
+Main tools:
+- [masurca](https://github.com/alekseyzimin/masurca)
+- [polca](https://github.com/alekseyzimin/masurca#polca)
+- [Chromosome scaffolder](https://github.com/alekseyzimin/masurca#chromosome-scaffolder)
+- [SAMBA](https://github.com/alekseyzimin/masurca#samba-scaffolder)
+
+Other tools:
+- [bwa](http://bio-bwa.sourceforge.net/) (used by polca)
```

Comment:
Any reason why the link isn't `https://github.com/lh3/bwa`?

That sourceforge.net site hasn't been updated since 2010-02-28 (at bottom of page)

Reply:
That's a good question. I'm going to fix it.